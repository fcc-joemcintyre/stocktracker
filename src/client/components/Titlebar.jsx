/**
 * Copyright (c) Joe McIntyre, 2016
 * license: MIT (https://github.com/fcc-joemcintyre/stocktracker/LICENSE.txt)
 */
import React from "react";

/**
 * Simple titlebar
 */
export default class Titlebar extends React.Component {
  render() {
    return (
      <div className="titleArea">
        <div className="titleText">Stock Tracker</div>
      </div>
    );
  }
}
