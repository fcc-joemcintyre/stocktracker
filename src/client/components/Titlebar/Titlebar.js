import React from 'react';

/*
 * Simple titlebar
 */
export const Titlebar = () => (
  <div className='titleArea'>
    <div className='titleText'>Stock Tracker</div>
  </div>
);
