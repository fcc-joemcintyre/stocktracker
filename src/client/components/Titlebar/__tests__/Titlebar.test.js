/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import { Titlebar } from '../Titlebar';

test ('Titlebar displays its content', () => {
  const component = renderer.create (
    <Titlebar />
  );
  const tree = component.toJSON ();
  expect (tree).toMatchSnapshot ();
});
