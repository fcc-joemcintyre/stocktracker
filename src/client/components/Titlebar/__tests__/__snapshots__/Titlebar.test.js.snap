// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Titlebar displays its content 1`] = `
<div
  className="titleArea"
>
  <div
    className="titleText"
  >
    Stock Tracker
  </div>
</div>
`;
